### PR TITLE
Add AllergyIntolerance resource  documentation for retrieve by allerg…

### PR DESCRIPTION
**Pull request by davischristi**
…y id.

Update documentation for AllergyIntolerance resource for retrieve by allergy id changes.

### CODE REVIEW

File: content/dstu2/allergy-intolerance.md [hunk 1]
```diff
@@ -19,15 +19,31 @@ Search for AllergyIntolerances that meet supplied query parameters:
 
 ### Parameters
 
- Name      | Required? | Type          | Description
------------|-----------|---------------|-----------------------------------------------------------------
- `patient` | Y         | [`reference`] | Who the sensitivity is for. Example: `12345`
- `status`  | N         | [`token`]     | [Certainty of the allergy or intolerance]. Example: `confirmed`
+ Name      | Required?                                                          | Type          | Description
+-----------|--------------------------------------------------------------------|---------------|-----------------------------------------------------------------------
+ `_id`     | This or patient, if populated all other parameters are not allowed | [`token`]     | The logical resource id associated with the resource. Example: `12345`
```
Comment (daliboz) [inline content/dstu2/allergy-intolerance.md]: use tick marks around patient so it's stylized (recognized as a field name) like the _id parameter for patient.

**Comment by daliboz** [comment]
Otherwise +1

**Comment by cerbs** [comment]
Needs to be rebased and changes to lib/dstu2_resources.rb moved to the appropriate example file

**Comment by cerbs** [comment]
👍  after updates

**Comment by cerbs** [comment]
Merged and squashed with deaf419

**Pull request closed without merging**